index.html

```diff
@@ -95,6 +95,28 @@ <h3>RTP header control</h3>
       </p>
     </section>
   </section>
+  <section id="rtcpeerconnection-interface">
+    <h3>
+      {{RTCPeerConnection}} extensions
+    </h3>
+    <p>
+      The {{RTCPeerConnection}} interface is defined in [[WEBRTC]]. This document
+      extends that interface by using Content-Security-Policy for ICE candidate
+      filtering.
+    </p>
+    <section>
+      <h2>Modifications to existing procedures</h2>
+
+      <p>Append the following paragraph to the
+      <a data-cite="WEBRTC#dfn-administratively-prohibited">administratively prohibited</a>
+      algorithm:</p>
+
+      <p>If <a data-cite="WEBAPPSEC-CSP#should-block-rtc-connection">
+        should-block-rtc-connection</a> on the current global object returns
+      "Blocked", all candidates MUST be <a data-cite="WEBRTC#dfn-administratively-prohibited">
```

Comment:
Thanks @zenhack for your patience! — I've modified the cite to get it to link (should work once TR updates), and I've specified the required argument. If this LGTY I'll merge it.
```suggestion
      <p>If <a data-cite="CSP#should-block-rtc-connection">
        should RTC connections be blocked for global?</a> with the
      [=relevant global object=] of the {{RTCPeerConnection}} object in question
      returns `"Blocked"`, then all candidates MUST be <a
        data-cite="WEBRTC#dfn-administratively-prohibited">
```